templates/cc/wrapper.go

```diff
@@ -6,5 +6,7 @@ const wrapperTpl = `
 	if ({{ hasAccessor . }}) {
 		const auto wrapped = {{ accessor . }};
 		{{ render (unwrap . "wrapped") }}
-	}
+	} {{ if .MessageRules.GetRequired }} else {
```

Comment:
I know we originally discussed doing the enforcement here, but I realized there's a required.go file where we could make the exact same update as we've done for .cc and .java. Thoughts?

Reply:
This would keep the implementation consistent.

Reply:
Discussed offline that we need to have it in both places, one for wrappers and one for messages.